pkg/storage/ibmcos/powervs_regions.go

```diff
@@ -0,0 +1,61 @@
+package ibmcos
+
+import (
+	"fmt"
+)
+
+// powerVSRegion describes respective IBM Cloud region and PowerVS zones associated with a region in PowerVS.
+// which helps to decide COS location from PowerVS region with the intention to keep COS in same region as PowerVS.
+type powerVSRegion struct {
+	description string
+	cosRegion   string
+}
+
+// powerVSRegions provides a mapping between Power VS and IBM Cloud regions.
+var powerVSRegions = map[string]powerVSRegion{
```

Comment:
Is there any way this map can be built dynamically? What happens when IBM adds a new region?

Reply:
no, unfortunately. we explored that when we created it for IPI. we have to keep up with this ourselves. i submitted a feature request to have some of this made available via an API but it's not been taken up.

Reply:
Thanks for expanding on this. 
With this code, will cluster install fail on a region that is not listed in this map?
We have this situation on aws (whenever they add a new region we have to upgrade their sdk or else install fails on that region), I'm trying to avoid having the same thing happen for IBM COS. Is there any fallback we could choose? Or would it just not work (in which case it is better to fail).

Reply:
> With this code, will cluster install fail on a region that is not listed in this map?
Yes, we would want cluster installation to get fail if an unknown region is passed.

> whenever they add a new region we have to upgrade their sdk or else install fails on that region
That's true, we are planning to keep this in common pkg like library-go and use it here in future(planning to do it after 4.13). So no need to update in this repo whenever new region is added.

Reply:
okay, so no fallback then.
it would indeed be a lot better to import this from library-go or some other shared package 👍🏼